cmd/query/app/http_handler.go

```diff
@@ -193,6 +195,42 @@ func (aH *APIHandler) getOperationsLegacy(w http.ResponseWriter, r *http.Request
 	aH.writeJSON(w, r, &structuredRes)
 }
 
+func (aH *APIHandler) transformOTLP(w http.ResponseWriter, r *http.Request) {
```

Comment:
* we have unit tests for all API endpoints, please add one for /transform
* similar to my comment in the UI PR (https://github.com/jaegertracing/jaeger-ui/pull/2145#discussion_r1472233464), I would suggest saving an OTLP test fixture file, and the corresponding UI JSON, and implementing submit/validate in the unit test

Reply:
Is it ok to have a json file in cmd/query/app/fixture and read that file in the .._test.go file and pass it to the test http server? None of the API endpoints seem to be reading a file and hence this question

Reply:
Yes. Probably because other tests are using dynamically generated data. Here it's important to use the input exactly as produced by otel collector.

Reply:
I have written a basic unit test for the API in a new file handler_otlp_translator_test.go